plugins/src/opengrok/auth/plugin/ldap/LdapFacade.java

```diff
@@ -89,6 +96,11 @@
      */
     private List<LdapServer> servers = new ArrayList<>();
 
+    /**
+     * server hooks
+     */
+    Hooks hooks;
```

Comment:
i know it doesnt make any real difference but all other members have their 'private' specified